### PR TITLE
Fix remote browsing language label display in search chips

### LINKED ISSUE #10989: Explore libraries - The applied language filter and the 'Clear all' option are missing
## Observed behavior
See the attached video.

## Expected behavior
Both the applied language filter and the 'Clear all' option should be available.

## Steps to reproduce the issue
1. Install the build asset from the following [PR](https://github.com/learningequality/kolibri/pull/10986)
2. Navigate to Library > Other libraries > Kolibri Library > Explore
3. Filter by language

## Video

https://github.com/learningequality/kolibri/assets/79847249/a880a1fe-9425-46c8-b0c7-75d3a34a7d94

## Usage Details
Kolibri 0.16alfa20
Windows 10, Ubuntu - Chrome

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary
* Stores fetched language filters from remotes in the global languages map
* Tracks the loading state of global label fetching
* Includes this loading state in the overall 'searchLoading' state to prevent premature display of search results before the search has finished loading

## References
Fixes [#10989](https://github.com/learningequality/kolibri/issues/10989)

## Reviewer guidance
Browse Kolibri Studio
Filter by a language that is not available on your local device
Observe properly rendered search chips
Filter by Uncategorized - see similarly.

Screenshot from 2023-07-18 12-49-00

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by pcenov — APPROVED** [review]
LGMT, the chips are displayed correctly now!

**Review by marcellamaki — APPROVED** [review]
lgtm too! good to go once merge conflicts are resolved

**Pull request merged**
